CHANGELOG.md

```diff
@@ -85,6 +85,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated the default mode for `CartesiaTTSService` and
   `CartesiaHttpTTSService` to `sonic-2`.
 
+### Removed
+
+- Removed deprecated `audio.resample_audio()`.
```

Comment:
Would it be worth to add here what they should use instead ? Like: `use 'create_default_resampler()' instead`

Reply:
The same applies to all the others that we have removed, in case we think this is a worthwhile approach.

Reply:
Done!